### PR TITLE
Questions from Zia

### PR DESCRIPTION
**Pull request by 2024zia**
I added my research questions please check.

### CODE REVIEW

**Comment by MurtazaHakimi** [comment]
Hallo Zia, 
many thanks for your efforts. your questions are really deep and we can at the end so much learn from them. but please check your 3. and 4. questions are just copied. you can remove one of them. 

regards
Murtaza

File: README.md [hunk 1]
```diff
@@ -21,3 +21,10 @@
 1. Which programming, scripting, and markup languages are used to develop programs in the last years?
 2. What are the most utilized database environments for development projects in the past year?
 3. Is there a discernible correlation between the size of companies and the adoption of specific AI-powered search tools for business operations?
+
+### Zia
+1. Are professional developers using artificial intelligence tools in their development process ?
+2. What is the biggest benefit that developers perceive from AI tools, and what are the secondary benefits that are closely associated with them?
+3. What is the biggest benefit that developers perceive from AI tools, and what are the secondary benefits that are closely associated with them?
```
Comment (MurtazaHakimi) [inline README.md]: your 2nd and 3. questions are just duplicated. please delete one of them.

Comment (2024zia) [reply README.md]: Thanks a lot for your comment ! I will fix it.

Comment (MurtazaHakimi) [reply README.md]: Well Done Zia.

**Comment by MurtazaHakimi** [comment]
Well done Zia, you did all the best.

**Pull request merged**